internal/cluster/distribution/eks/eksprovider/workflow/amazon.go

```diff
@@ -49,15 +49,23 @@ const asgFulfillmentWaitInterval = asgWaitLoopSleep
 
 // getStackTags returns the tags that are placed onto CF template stacks.
 // These tags  are propagated onto the resources created by the CF template.
-func getStackTags(clusterName, stackType string) []*cloudformation.Tag {
-	return append([]*cloudformation.Tag{
+func getStackTags(clusterName, stackType string, customTagsMap map[string]string) []*cloudformation.Tag {
+	tags := append([]*cloudformation.Tag{
 		{Key: aws.String("banzaicloud-pipeline-cluster-name"), Value: aws.String(clusterName)},
 		{Key: aws.String("banzaicloud-pipeline-stack-type"), Value: aws.String(stackType)},
 	}, internalAmazon.PipelineTags()...)
+
+	for k, v := range customTagsMap {
```

Comment:
This could result in builtin tags being overwritten IMHO.

Reply:
add custom tags first